packages/tpetra/core/src/Tpetra_Details_FixedHashTable_decl.hpp

```diff
@@ -328,7 +328,7 @@ class FixedHashTable {
     // numPairs_ field (remembering the size of the input array of
     // keys).
     if (this->hasContiguousValues ()) {
-      return val_.extent (0) + static_cast<offset_type> (lastContigKey_ - firstContigKey_);
+             return val_.extent (0) + static_cast<offset_type> (lastContigKey_ - firstContigKey_);
```

Comment:
```suggestion
      return val_.extent (0) + static_cast<offset_type> (lastContigKey_ - firstContigKey_);
```
(no need to touch this line I guess - especially that it would break the chosen file tab size)